lib/Auth/Process/ΕvaluateΕntitlement.php

```diff
@@ -0,0 +1,175 @@
+<?php
+
+/**
+ * Attribute filter to evaluate entitlement for access to Services.
```

Comment:
"Attribute filter for evaluating the resources capabilities of the authenticating user. 
A capability defines the resource or child-resource the user is allowed to access, optionally specifying certain actions the user is entitled to perform. Capabilities can be used to convey – in a compact form – authorisation information. Capability values are formatted as URNs following the syntax specified in in https://aarc-community.org/guidelines/aarc-g027"